### DISCUSSION TITLE
Remembering Output Format

### DISCUSSION THREAD

**Discussion by Glenn1963** [mifi/lossless-cut#1196] [Q&A] [answered]
Is the app supposed to be remembering or is there a way to make it always default to my chosen output format?
Thanks!

(If this has an answer, I couldn't find it, sorry.)

**Comment by mifi** [comment, marked as answer]
the lock icon next to format

**Reply by Glenn1963** [reply]
Oof, thank you! I've been setting it each time in the dialog that appears after I click Export. Never noticed that Lock. My bad, appreciate the help :)
